sdccc/src/main/java/com/draeger/medical/sdccc/tests/glue/direct/FailingHttpHandler.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * This Source Code Form is subject to the terms of the MIT License.
+ * Copyright (c) 2022 Draegerwerk AG & Co. KGaA.
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+package com.draeger.medical.sdccc.tests.glue.direct;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.eclipse.jetty.http.HttpStatus;
+import org.somda.sdc.dpws.http.HttpException;
+import org.somda.sdc.dpws.http.HttpHandler;
+import org.somda.sdc.dpws.soap.CommunicationContext;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+/**
+ * HTTPHandler that fails reception (answers the Request 500 Return Code)
```

Comment:
"answers the Request 500 Return Code" -> "answers the request with the http response status code 500"

Reply:
Corrected.